src/api/admin.ts

```diff
@@ -26,6 +27,12 @@ app.post('/sslCerts', async (req, res) => {
 
     if (stderr) return res.json({ stderr })
 
+    const { stdout: ipaddress } = await exec('curl ifconfig.me')
+    const setRecords: Function = await providers[service].setRecord(
```

Comment:
I got a TS warning `Element implicitly has an 'any' type because expression of type 'any' can't be used to index type '{ dns_gd: Function; }'.ts(7053)`
I don't know how to solve it

Reply:
`setRecords` is not a function. it is data returned from the API call